_docs_v7/Solver-Setup.md

```diff
@@ -40,15 +49,111 @@ SU2 is capable of dealing with different kinds of physical problems. The kind of
+  - `SUSTAINING`: refers to the controlled decay correction.
+
+- `SA_OPTIONS`:
+  - `NONE`: refers to the modification where the $f_{t2}$ term is set to zero, i.e., $c_{t3} = 0$ (default).
+  - `NEGATIVE`: refers to the negative Spalart-Allmaras modification.
+  - `EDWARDS`: refers to the so-called Edwards modification.
+  - `WITHFT2`: refers to the standalone version.
+  - `QCR2000`: refers to the Quadratic Constitutive Relation modification, 2000 version.
+  - `COMPRESSIBILITY`: refers to the Mixing Layer Compressibility modification.
+  - `ROTATION`: refers to the rotation and curvature effects correction.
+  - `BCM`: refers to the SA-BCM transitional turbulence model.
+  - `EXPERIMENTAL`: Allow experimental combinations of options (according to [NASA TMR](https://turbmodels.larc.nasa.gov/)).
+
+We refer for example to [NASA TMR](https://turbmodels.larc.nasa.gov/) for a detailed explanation and definition of each correction and variation.
+
+#### Spalart-Allamaras ####
+The single transported Spalart-Allmaras variable $\tilde{\nu}$ is initialized with the value at the farfield or inlet boundary. As suggested in the literature, the value there is computed as $\tilde{\nu}/\nu=\text{turb2lam}$. In SU2 the free-stream Spalart-Allmaras variable to kinematic laminar viscosity ratio, $\text{turb2lam}$, is defined by the `FREESTREAM_NU_FACTOR` config file field. The default value is $\tilde{\nu}/\nu = 3.0$ avoiding laminar solutions.
+
+An extension of SU2 includes an hybrid turbulence model: the Spalart-Allmaras original model with Detached-Eddy Simulation (DES) modification. Refer to Eduardo Moina's thesis?. The use of the hybrid RANS/LES model is specified in the cofig file with the field `HYBRID_RANSLES`. Four different techniques are currently implemented:
+
+- `SA_DES` Detached-Eddy Simulation
+- `SA_DDES` Delayed Detached-Eddy Simulation
+- `SA_ZDES` Zonal Detached-Eddy Simulation
+- `SA_EDDES` Enhanced Detached-Eddy Simulation
+
+The DES constant can be controlledd by the field `DES_CONST`, with 0.65 as default.
+
+#### Menter’s k-omega SST Model ####
+As initial conditions, the values of are initialized at all grid point with the farfield values. The farfield conditions for the turbulent kinetic energy $k$ is computed as: $k = \frac{3}{2}(UI)$, whereby $U$ is the freestream velocity magnitude and $I$ is the turbulence intensity. This latter value can be specified in the config file by the `FREESTREAM_TURBULENCEINTENSITY` field, with defaul value to 0.05 which corresponds to a $5\%$. The freestream value of the turbulent frequency $\omega$ is computed as: $\omega = \frac{\rho k}{\mu}(\frac{\mu_t}{\mu})^{-1}$, where $\rho$ and $\mu$ are the freestream density and laminar viscosity respectively and $\frac{\mu_t}{\mu}$ is the turbulent viscosity ratio, also known as freestream dissipation. Its value can be specified in the `FREESTREAM_TURB2LAMVISCRATIO` field, defaulting to $10$.
```

Comment:
Would be nice if someone checks the boundary conditions for omega. I am not that familiar with the k-w model.